### PR TITLE
coppeliasim: Update to version 4.5.1 and fix autoupdate

### PR DESCRIPTION
**Pull request by StarsbySea**
Closes #XXXX

Relates to #XXXX

- [X] I have read the [Contributing Guide](https://github.com/ScoopInstaller/.github/blob/main/.github/CONTRIBUTING.md).

### CODE REVIEW

File: bucket/coppeliasim.json [hunk 1]
```diff
@@ -23,13 +23,13 @@
         ]
     ],
     "checkver": {
-        "url": "https://www.coppeliarobotics.com/helpFiles/en/versionInfo.htm",
-        "regex": "CoppeliaSim\\s+V([\\d.]+)\\s+\\((?!not yet)"
+        "url": "https://www.coppeliarobotics.com/downloads",
+        "regex": "CoppeliaSim_Edu_V([\\d_]+)_rev(?<rev>\\d+)_Win.zip"
```
Comment (HUMORCE) [inline bucket/coppeliasim.json:27]: We should not hardcode `revX`.

**Review by HUMORCE — CHANGES_REQUESTED** [review]

File: bucket/coppeliasim.json (hunk 1 above)
Comment (StarsbySea) [reply bucket/coppeliasim.json]: https://github.com/StarsbySea/Extras/blob/02454e4fc89dd864bf54d4242b1c5c241a49d47d/bucket/coppeliasim.json#L27

I have made some changes as per your request, please see.

But from this page https://www.coppeliarobotics.com/previousVersions, recent versions contain the rev field that's all.

Comment (HUMORCE) [reply bucket/coppeliasim.json]: `scoop install/download coppeliasim@4.3.0` broken when hardcode it.

**Review by HUMORCE — APPROVED** [review]

**Pull request merged**